lib/nexmo_developer/app/views/slack/join.html.erb

```diff
@@ -1,25 +0,0 @@
-<center>
```

Comment:
@bencgreenberg we need this view for `/community/slack`

Reply:
It's been moved to the complementary PR in NDP https://github.com/Nexmo/nexmo-developer/pull/3477

Reply:
aha, I didn't see that 🤦